pkg/grpc/config/config.proto

```diff
@@ -239,3 +241,8 @@ message Settings {
   optional string favicon_url = 90;
   optional string error_message_first_paragraph = 91;
 }
+
+message DownstreamMtlsSettings {
+  optional string ca = 1;
+  optional string crl = 2;
```

Comment:
there was some historic b64 strings / bytes confusion in the protobuf, I believe we wanted to use use actual `bytes` in protobuf? i.e. https://github.com/pomerium/pomerium/blob/bb2fb9730ad4f714d025578305236081fba32db9/pkg/grpc/config/config.proto#L136-L140